cmd/ddev/cmd/composer-create.go

```diff
@@ -69,18 +75,6 @@ project root will be deleted when creating a project.`,
 			}
 		}
 
-		// The install directory may be populated if the command has been
-		// previously executed using the same container.
-		output.UserOut.Printf("Ensuring temporary install directory in web container is empty")
-		installDir := "/var/www/html/.tmp-install"
-		_, _, err = app.Exec(&ddevapp.ExecOpts{
-			Service: "web",
-			Cmd:     []string{"sh", "-c", fmt.Sprintf("rm -rf %s", installDir)},
-		})
-		if err != nil {
-			util.Failed("Failed to create project: %v", err)
-		}
-
 		// Remove any contents of project root
 		util.Warning("Removing any existing files in project root")
```

Comment:
This isn't actually true is it? We *overwrite* but don't currently remove any existing files otherwise? Maybe the warning should be "Existing files may be overwritten" or perhaps we should actually delete everything but .ddev.

Reply:
We are explicity removing everything except .ddev in /var/www/html here, though.

Reply:
Ah, not we're definitely not removing everything. It does not remove dot-files or dot-directories. But I see it does everything else. (I did lose track of context, but also this is not removing like the old failed composer installs, which isn't a big deal). But it will be much improved by doing this on the host.